### PR TITLE
Migrate to TS: eachHourOfInterval

### PR DESCRIPTION
**Pull request by minitesh**

### CODE REVIEW

File: src/eachHourOfInterval/test.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ describe('eachHourOfInterval', () => {
     assert.throws(block, RangeError)
   })
 
-  it('throws an exception if the interval is undefined', () => {
+  it.only('throws an exception if the interval is undefined', () => {
```
Comment (kossnocorp) [inline src/eachHourOfInterval/test.ts:87]: Could you please remove it?

File: src/eachHourOfInterval/index.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ import requiredArgs from '../_lib/requiredArgs/index'
  * //   Mon Oct 06 2014 15:00:00
  * // ]
  */
-export default function eachHourOfInterval(dirtyInterval, options) {
+export default function eachHourOfInterval(dirtyInterval: Interval, options?: {step?: number}): Date[] {
```
Comment (kossnocorp) [inline src/eachHourOfInterval/index.ts:35]: I don't see `Interval` import.

**Review by kossnocorp — CHANGES_REQUESTED** [review]
Please import `Interval` correctly and remove `only` from tests.

File: src/eachHourOfInterval/test.ts (hunk 1 above)
Comment (minitesh) [reply src/eachHourOfInterval/test.ts:87]: Yeah sorry my bad.
I realized it later on after I switched off my laptop :(
Will do the changes

File: src/eachHourOfInterval/index.ts (hunk 2 above)
Comment (minitesh) [reply src/eachHourOfInterval/index.ts:35]: There is no need to import.
There is a definition in `typings.d.ts`
```
type Interval = {
  start: Date | number
  end: Date | number
}
```

Comment (kossnocorp) [reply src/eachHourOfInterval/index.ts:35]: The thing is that typings.d.ts is automatically generated and won't be a thing in the future, so we better use explicit imports. No worries, though, I will do it myself in the main branch.

**Review by kossnocorp — APPROVED** [review]
Thanks!

File: src/eachHourOfInterval/index.ts (hunk 2 above)
Comment (minitesh) [reply src/eachHourOfInterval/index.ts:35]: Ok.
I will take care of this in my next PRs.

**Pull request merged**